crates/rust-analyzer/src/lib.rs

```diff
@@ -115,35 +117,59 @@ fn completion_item_hash(item: &CompletionItem, is_ref_completion: bool) -> [u8;
         u8::from(item.deprecated),
         u8::from(item.trigger_call_info),
     ]);
+
+    hasher.update(item.label.primary.len().to_le_bytes());
```

Comment:
`to_ne_bytes` would be enough, we don't persist these for a long time, and if CI passes, we're not checking for specific hash values in tests.

Reply:
There was one place that was already using `to_le_bytes`, so I decided to follow suite.  But I agree `to_ne_bytes` makes more sense in this case.

Do you want me to change the one existing case of `to_le_bytes` as well, or leave it alone?

Reply:
Sure, let's use `to_ne_bytes`.